Library link configuration in database

aslagle opened the pull request.

leonardr commented on api/opds.py:
```diff
@@ -437,17 +452,12 @@ def annotate_feed(self, feed, lane):
-                d = dict(href=value, type="text/html", rel=rel)
+    def add_configuration_links(self, feed):
+        for rel in self.CONFIGURATION_LINKS:
+            _db = Session.object_session(self.library)
```
Call this outside the for loop.

leonardr commented on migration/20170605-move-configuration-links-into-db.py:
```diff
@@ -0,0 +1,36 @@
+        ("copyright", Configuration.acknowledgements_url()),
+        ("about", Configuration.about_url()),
+        ("license", Configuration.license_url()),
+):
```
If you use the code these methods use, you can remove the methods themselves in this branch; otherwise we need to wait until everyone is migrated.

leonardr commented:
I have a couple suggestions but this is :+1: as is.

Looking at this code, I realize I was implicitly assuming that we would start allowing administrators to paste in the actual HTML they wanted to use as "about", etc., and then serve that HTML from special controllers. I still think we should allow that as an option, but it's a much bigger task than just moving the links out of the configuration, so let's not do it now.

aslagle commented:
I updated the migration and removed the old config methods in https://github.com/NYPL-Simplified/server_core/pull/516, and also fixed another place that used them.
